streaming/text/convert/pile.py

```diff
@@ -85,8 +88,8 @@ def parse_args() -> Namespace:
         type=str,
         default='zstd:16',
         help='Compression algorithm to use. Empirically, Zstandard has the best performance in ' +
-        'our benchmarks. Tune the compresion level (from 1 to 22) to trade off time for ' +
-        'quality. Default: zstd:16',
+        'our benchmarks. Tune the compression level (from 1 to 22) to trade off time for ' +
+        'quality. Defaults to ``zstd:16``',
```

Comment:
double ticks in help strings

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/171/commits/00c1e6d5f746940e20491297a395cc00e041ea37.